source/lex.h

```diff
@@ -366,25 +365,8 @@ auto expand_string_literal(
     auto first_quote_pos = pos;
```

Comment:
This variable is no longer needed, right? Seems to be unused now

Reply:
Yes, true. Sorry for not paying attention to details.

Reply:
Fixed